### PR TITLE
chore: App Sidebar testing

### PR DESCRIPTION
**Pull request by hetunandu**
Updates the Cypress testing suite to handle the new app sidebar as a new way to navigation

fixes: #28388

### CODE REVIEW

File: app/client/cypress/support/Pages/AggregateHelper.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import type CodeMirror from "codemirror";
 import { ReusableHelper } from "../Objects/ReusableHelper";
 import type { EntityItemsType } from "./AssertHelper";
 import { EntityItems } from "./AssertHelper";
+import EditorNavigation, { SidebarButton } from "./EditorNavigation";
```
Comment (ankitakinger) [inline app/client/cypress/support/Pages/AggregateHelper.ts:8]: Are we using this in the file?

File: app/client/cypress/support/Pages/EditorNavigation.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  Pages = "Pages",
+  Libraries = "Libraries",
+  Settings = "Settings",
+}
+class EditorNavigation {
+  locators = {
+    sidebar: ".t--sidebar",
+    sidebarButtons: (name: SidebarButton) => `.t--sidebar-${name}`,
+  };
+  sidebar(button: SidebarButton) {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/Pages/EditorNavigation.ts:12]: What is sidebar here? method to Navigate to one of side bar item? Can we please name it appropriately?

Comment (Aishwarya-U-R) [reply app/client/cypress/support/Pages/EditorNavigation.ts:12]: Something like NavigateToSidebar() - and TS helper methods are CamelCase please.

Comment (hetunandu) [reply app/client/cypress/support/Pages/EditorNavigation.ts]: Signature now is `EditorNavigation.ViaSidebar()`

**Pull request merged**